src/courtroom.cpp

```diff
@@ -6760,6 +6761,17 @@ void Courtroom::handle_clock(QString time)
   ui_rp_clock->show();
 }
 
+void Courtroom::format_clock(QStringList time_format) {
```

Comment:
Why is this function responsible for parsing the arguments? Pass the arguments here and handle the QStringList unpacking in packet_distribution.cpp